javascript/CMSPageHistoryController.js

```diff
@@ -156,10 +156,16 @@
 			 * Function: _unselect()
 			 *
 			 * Unselects the row from the form selection.
+			 *
+			 * Using regular js to update the class rather than this.removeClass('active')
+			 * because the latter causes the browser to continuously call
+			 * element.compareDocumentPosition, causing the browser to hang for long
+			 * periods of time, especially on pages with lots of versions (e.g. 100+)
 			 */
 			_unselect: function() {
-				this.removeClass('active');
-				this.find(":input[type=checkbox]").attr("checked", false);
+				var tr = this.get(0);
+				tr.className = $.trim(tr.className.replace(/\bactive\b/, ''));
```

Comment:
`Element.classList` support is pretty common these days, so you could use it like so:

```js
this.get(0).classList.remove('active');
```

Reply:
Only IE11 according to https://caniuse.com/#feat=classlist

Reply:
IE 10 and 11 😄

Reply:
Ah, yes. So not quite enough. Thanks for the suggestion anyway. We could use that in 4.x instead.

Reply:
merged up into 4.x using classList